flang/include/flang/Optimizer/HLFIR/Passes.td

```diff
@@ -13,6 +13,9 @@ include "mlir/Pass/PassBase.td"
 def ConvertHLFIRtoFIR : Pass<"convert-hlfir-to-fir", "::mlir::ModuleOp"> {
   let summary = "Lower High-Level FIR to FIR";
   let constructor = "hlfir::createConvertHLFIRtoFIRPass()";
+  let dependentDialects = [
```

Comment:
nit: Is this necessary?

Reply:
Yep, the added lit test would not work without it.  It does not have `func.func`, so the `FuncDialect` is not implicitly loaded. The pass inserts a new `FuncOp`, though, with the dialect being loaded.

Reply:
Ok I see! Thanks for the explanation.